content/docs/client-options.md

```diff
@@ -53,7 +54,7 @@ These clients are compatible with our [staging endpoint for ACME v2](https://com
 - [Get HTTPS for free](https://gethttpsforfree.com)
 - [ZeroSSL](https://ZeroSSL.com) (Fully in-browser process, inc. CSR generation)
 - [Certificate Automation](https://www.certificateautomation.com/)
-- [SSL for free](https://www.sslforfree.com/) (Fully in-browser process, inc. CSR generation)
+- [SSL for free](https://www.sslforfree.com/) (Fully in-browser process, inc. CSR generation, wildcard certificate available)
```

Comment:
So far we've avoided updating the ACMEv1 entries to also mention they are ACMEv2/wildcard compatible.

Can you please remove this part of your diff?